assets/javascripts/openqa.js

```diff
@@ -161,7 +161,7 @@ function renderList(items) {
     var ul = document.createElement('ul');
     items.forEach(function(item) {
         var li = document.createElement('li');
-        li.appendChild(document.createTextNode(item));
+        li.innerHTML = item;
```

Comment:
HTML is generally slower. Why would you make this change?

Reply:
In order to be able to display a link to the documentation. I doubt the performance difference will be notable.